package-lock.json

```
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.70.1",
+      "version": "0.70.17",
```

Package version back to the one in the base branch! 

```suggestion
      "version": "0.70.1",
```

---

Corrected, after a couple of tries. Sorry that took me so long to get right, but now it registers as the addition of the profile json as the only difference from the main branch.